test/e2e/kubernetes/testdriver-1.25.yaml

```diff
@@ -21,6 +21,7 @@ DriverInfo:
     controllerExpansion: true
     nodeExpansion: true
     onlineExpansion: false
+    offlineExpansion: true
```

Comment:
Is this required by the upgrade?

Reply:
This file list available capabilities of the csi-driver that the kubernetes csi test suite will verify. We can only specify capabilities that are added in the lowest-tested version (at least with the current setup). This PR moves this from 1.23 to 1.25. The only capability added is `offlineExpansion`, but this already defaulted to `true`, so everything should keep working as expected.